mandatory/4-tax.js

```diff
@@ -5,19 +5,15 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
-
-/*
-  CURRENCY FORMATTING
-  ===================
-  The business has informed you that prices must have 2 decimal places
-  They must also start with the currency symbol
-  Write a function that adds tax to a number, and then transforms the total into the format £0.00
-
-  Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
-*/
-
-function addTaxAndFormatCurrency() {}
+function calculateSalesTax(priceBeforetax) {
+let priceWithTax = priceBeforeTax * 1.2;
```

Comment:
Indentation error! Lines 9 and 10 should be indented because they are in a code block.